testcases/unit/pintest.c

```diff
@@ -22,10 +22,20 @@ int main(int argc, char *argv[])
 	switch(getopt(argc, argv, "pn")) {
 	case 'p':
 		pin = pin_prompt(&buf, "Enter the PIN: ");
+#if defined(_AIX)
+		/* AIX prints an empty string instead of Linux-style (null) */
+		if (pin == NULL)
+			pin = "(null)";
+#endif
 		printf("pin: %s\n", pin);
```

Comment:
Maybe this would be shorter:

printf("pin: %s\n", pin != NULL ? pin : "(null)");

You can do this unconditionally, won't hurt for non-AIX systems either. 

Same below.